README.md

```diff
@@ -49,7 +49,7 @@ first, and then:
     conda activate myenv
 
     # Install the stable release.
-    conda install -c theochem procrustes
+    conda install -c theochem/label/dev qc-procrustes
```

Comment:
@fwmeng88, we will be able to use the `-c theochem` again upon making a `main` release (as opposed to `dev` release we have now), which will be hopefully soon.

Reply:
I will update it once we have the main release. At least, people can install with `conda` and `pip` at this moment. Thanks for the clarification. @FarnazH